src/nfts/slice.ts

```diff
@@ -0,0 +1,41 @@
+import { createSlice } from '@reduxjs/toolkit'
+import { Nft } from 'src/nfts/types'
+
+export interface FetchMyNftsCompleted {
+  nfts: Nft[]
+}
+
+export interface State {
+  myNftsLoading: boolean
```

Comment:
i think we should remove "my" from all the variable names? we don't have "fetchMyTokens" etc.

Reply:
I wasn't sure if we'd eventually store other Nfts in redux, but we can migrate the redux state if needed. I'll remove all these _My_ prefixes.